weavertest/multi.go

```diff
@@ -116,26 +115,17 @@ func initMultiProcess(ctx context.Context, t testing.TB, config string) weaver.I
 	}
 
 	// Create and run an envelope for the main process.
-	options := envelope.Options{
-		Restart:         envelope.Never,
-		Retry:           retry.DefaultOptions,
-		GetEnvelopeConn: func() *conn.EnvelopeConn { return econn },
-	}
-	e, err := envelope.NewEnvelope(weaveletInfo, dep.App, b, options)
+	e, err := envelope.NewEnvelope(weaveletInfo, dep.App, b, envelope.Options{})
```

Comment:
Hm, maybe I'm missing something, but what if we don't make an envelope here at all? Then, we can unexport the `SetConn` method again while keeping the `GetEnvelopeConn` option deleted?

Reply:
haha, oh ya! I missed that after this change `Envelope` is doing nothing! Removing it entirely seems to work, I'll push a commit that makes that change now.